Mage.Sets/src/mage/cards/g/GarrukSavageHerald.java

```diff
@@ -39,15 +45,15 @@ public GarrukSavageHerald(UUID ownerId, CardSetInfo setInfo) {
         this.addAbility(new LoyaltyAbility(new GarrukSavageHeraldEffect(), 1));
 
         // −2: Target creature you control deals damage equal to its power to another target creature.
-        DamageWithPowerFromOneToAnotherTargetEffect effect = new DamageWithPowerFromOneToAnotherTargetEffect();
+        Effect effect = new DamageWithPowerFromOneToAnotherTargetEffect();
         effect.setText("Target creature you control deals damage equal to its power to another target creature");
 
         Ability minusAbility = new LoyaltyAbility(effect, -2);
         TargetControlledCreaturePermanent controlledCreature = new TargetControlledCreaturePermanent();
         controlledCreature.setTargetTag(1);
         minusAbility.addTarget(controlledCreature);
 
-        FilterCreaturePermanent filter = new FilterCreaturePermanent("Another creature: damage dealt to");
```

Comment:
As info: there is special method in target to enable extra message in choose dialog for selecting target -- `target.withChooseHint("some hint message")`
![shot_200629_072931](https://user-images.githubusercontent.com/8344157/85970184-b0e49180-b9da-11ea-865a-03199702d46a.png)

Reply:
fixed